templates/legal/nvidia-jetson-and-ubuntu.html

```diff
@@ -0,0 +1,21 @@
+{% extends "legal/_base_legal.html" %}
```

Comment:
Looks like the base template does not support the title block which is should. Could you fix that in this PR?

Reply:
I can see it in the `head` of the document and it appears on hover on the tab, where else is it supposed to be seen?

Reply:
Sorry your right.